ot/mapping.py

```diff
@@ -173,7 +173,7 @@ def nearest_brenier_potential_fit(
                         - c3 * (G[j] - G[i]).T @ (X[j] - X[i])
                     ]
         problem = cvx.Problem(objective, constraints)
-        problem.solve(solver=cvx.ECOS)
+        problem.solve()
```

Comment:
could be good to allow the solver to be passed in parameter? My experience anis that it can have a large impact on performance

Reply:
sounds good, and we'll leave the default as the cvxpy default